cloud/jenkins/ps_operator_gke_version.groovy

```diff
@@ -360,7 +400,7 @@ pipeline {
                     source $HOME/google-cloud-sdk/path.bash.inc
                     gcloud auth activate-service-account alpha-svc-acct@"${GCP_PROJECT}".iam.gserviceaccount.com --key-file=$CLIENT_SECRET_FILE
                     gcloud config set project $GCP_PROJECT
-                    gcloud alpha container clusters delete --zone us-central1-a $CLUSTER_NAME-basic | true
+                    gcloud alpha container clusters list --format='csv[no-heading](name)' --filter $CLUSTER_NAME | xargs gcloud alpha container clusters delete --zone $GKERegion --quiet || true
```

Comment:
from log:
```
+ gcloud alpha container clusters list '--format=csv[no-heading](name)' --filter jenkins-par-psmo-3eb5d9a
+ xargs gcloud alpha container clusters delete --zone --quiet
ERROR: (gcloud.alpha.container.clusters.delete) argument --zone/-z: expected one argument
Usage: gcloud alpha container clusters delete NAME [NAME ...] [optional flags]
  optional flags may be  --async | --help | --region | --zone

For detailed information on this command and its flags, run:
  gcloud alpha container clusters delete --help
+ true
```
you do not have` $GKERegion`

Reply:
fixed